src/validators/function.rs

```diff
@@ -16,6 +16,13 @@ use super::{build_validator, BuildContext, BuildValidator, CombinedValidator, Ex
 
 pub struct FunctionBuilder;
 
+fn destructure_function_schema(schema: &PyDict) -> PyResult<(&str, &PyAny)> {
```

Comment:
probably worth changing signature into
```suggestion
fn destructure_function_schema(schema: &PyDict) -> PyResult<(bool, &PyAny)> {
```
Then moving the `is_model_instance_method` logic in here.

Reply:
done in 04bc191